components/service/glean/src/test/java/mozilla/components/service/glean/TestUtil.kt

```diff
@@ -122,25 +124,11 @@ internal fun resetGlean(
     config: Configuration = Configuration(),
     clearStores: Boolean = true
 ) {
-    Glean.enableTestingMode()
-
     // We're using the WorkManager in a bunch of places, and Glean will crash
     // in tests without this line. Let's simply put it here.
     WorkManagerTestInitHelper.initializeTestWorkManager(context)
 
-    if (clearStores) {
-        // Clear all the stored data.
-        val storageManager = StorageEngineManager(applicationContext = context)
-        storageManager.clearAllStores()
-        // The experiments storage engine needs to be cleared manually as it's not listed
-        // in the `StorageEngineManager`.
-        ExperimentsStorageEngine.clearAllStores()
-    }
-
-    // Init Glean.
-    Glean.initialized = false
-    Glean.setUploadEnabled(true)
-    Glean.initialize(context, config)
+    TestUtils.resetGlean(context, config, clearStores)
```

Comment:
Please, let's not do this if possible :)

Reply:
I think we have to, both for initialization and clearing storage between the Sync and history tests. AFAICT, that's the only additional function we need Glean to expose, though.

Reply:
Mh, why do we need that? They are different pings, right? I'm happy to meet today to talk through this!

Reply:
I forgot I still have to send the ping after checking the values in `send{History, Bookmarks}Ping`. 🤦‍♀ With that fixed, the tests work! 😅